src/main/scala/scalismo/package.scala

```diff
@@ -38,6 +38,10 @@ package object scalismo {
       gc.SetScheduleTime(60, TimeUnit.SECONDS)
       gc.Start()
 
+      // we need to stop the garbage collection when the JVM shutdowns.
+      // Otherwise it prevents the JVM from shutting down correctly.
+      Runtime.getRuntime().addShutdownHook(new Thread { gc.Stop() })
```

Comment:
Huh? That's interesting. That would be an indicator that the shutdown hook doesn't get executed '''because''' the GC is still running? Or something similar...

Either way, I don't really understand how this version could work, then. From my understanding, it would immediately terminate the garbagecollector, and install a no-op shutdown hook.

I think this might require more investigation... I'll take a look at this maybe tomorrow.